src/canmatrix/tests/test_frame_decoding.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import canmatrix.formats
+import os.path
+import sys
+
+def test_decode_from_dbc():
+    here = os.path.dirname(os.path.realpath(__file__))
+    cm = canmatrix.formats.loadp(os.path.join(here ,"test_frame_decoding.dbc"), flatImport = True)
+    # 001#8d00100100820100
+    if sys.version_info > (3, 0):
```

Comment:
```suggestion
    if sys.version_info > (3,):
```
As is it treats 3.0 as py2.  But, tests seem to pass fine if we just always use `bytearray()`.  Some reason not to drop the version check?  (consider repeats below if change is made)

Reply:
good point.
I'll change to bytearray everything, than it's py2/py3 compatible.